app/src/main/java/com/sebastianvm/mango/ui/mvvm/BaseViewModel.kt

```diff
@@ -10,7 +10,7 @@ interface State
 interface UserAction
 
 abstract class BaseViewModel<S : State, A : UserAction>(
-    initialState: S,
+    initialState: S
```

Comment:
lol are our versions of ktlint battling it out for the trailing comma? 😂

Reply:
No, mine remove it too, but I hadn't committed it (hence this fix lol)